travis/samples/app-gradle/.yo-rc.json

```diff
@@ -32,6 +32,6 @@
     "messageBroker": false,
     "serviceDiscoveryType": false,
     "clientPackageManager": "yarn",
-    "clientFramework": "angular1"
+    "clientFramework": "angular2"
```

Comment:
since we do a lot of ejs conditions and stuff in gradle lets keep this build as ng1 as we already have ng2-jwt with gradle. may be we can rename that accordingly to reflect that and also rename the options changed here to show ng2 in the name